packages/iris-grid/src/IrisGridUtils.ts

```diff
@@ -98,7 +98,7 @@ export type DehydratedUserColumnWidth = [ColumnName, number];
 export type DehydratedUserRowHeight = [number, number];
 
 export type DehydratedSort = {
-  column: ModelIndex;
+  column: ColumnName | ModelIndex;
```

Comment:
I think we should make this `LegacyDehydratedSort` or something since moving forward we don't want `DehydratedSort` to use `ModelIndex` as the column. Then `hydrate` can take `LegacyDehydratedSort` and `dehydrate` return `DehydratedSort` without the `| ModelIndex`

Reply:
@mattrunyon I've added the `LegacyDehydratedSort` interface as suggested